Week-3/Ex-1/index.js

```diff
@@ -0,0 +1,30 @@
+
+function memoize(fn) {
+    let cache = new Map();
+    return function(...args) {
+        const key = args.toString();
+        console.log(`key formed is: ${key}`);
```

Comment:
Too many consoles, when raising PR try to remove the consoles which are not being used

Reply:
Noted.